bHY6yh-de2985/cod-bxxqqq_bHY6yh-de2985.xml

```diff
@@ -0,0 +1,195 @@
+            <p xml:id="bHY6yh-de2985-d1e114">
+              De verbo dei et 
+              <lb ed="#B"/>exauditione eius. Rubrica. <sic>II</sic><!--there are unambigious tittles on them--> 
+    </p>
+            <p xml:id="bHY6yh-de2985-d1e121">
+              <lb ed="#B"/>
+              <ref xml:id="bHY6yh-de2985-Rd1e118">/ In leuitico capitulo 26 et</ref> <quote xml:id="bHY6yh-de2985-Qd1e125">penulti Si in
+              <lb ed="#B"/>preceptis meis ambulaueritis 
+              <lb ed="#B"/>et mandata mea custodieritis <sic>etfe<lb ed="#B" break="no"/>ceritis</sic> 
+              ea. dabo vobis pluuias temporibus
+              <lb ed="#B"/>suis et terra gignet germen su<lb ed="#B" break="no"/>um 
+              et pomis arbores replebuntur 
+              <lb ed="#B"/>apprehendet messium tritura vin<lb ed="#B" break="no"/>demiam: 
+              et vindemia occupabitur 
+              <lb ed="#B"/>sementem et commedetis panem vestrum 
+              <lb ed="#B"/>insaturitate et absque pauore 
+              <lb ed="#B"/>habitabitis in terra vestra et dabo pacem 
+              <lb ed="#B"/>finibus vestris dormietis et non
+              <lb ed="#B"/>erit qui exterreat Auferam 
+              <lb ed="#B"/>malas bestias et gladius non 
+              <lb ed="#B"/>transibit terminos vestros <subst><add place="margin-right">persequemini inimico usus:</add></subst> et corru<lb ed="#B" break="no"/>ent 
+              coram vobis. </quote>
+            </p>
+            <p xml:id="bHY6yh-de2985-d1e163">
+              <lb ed="#B"/>
+                <ref xml:id="bHY6yh-de2985-Rd1e161">Moyses in deuteronomio <!--line jumps up--> capitulo 5</ref> <quote xml:id="bHY6yh-de2985-Qd1e167">Non declines <!--line jumps down--> neque ad dexte<lb ed="#B" break="no"/>ram 
+                neque ad siuistram sed per viam
+                <lb ed="#B"/>quam praecipit vobis deus ambula<lb ed="#B" break="no"/>bitis 
+                vt viuatis et bene sit vobis 
+                <lb ed="#B"/>et procellentur dies in terra <unclear>possi<lb ed="#B" break="no"/>onis</unclear> vestre. </quote>
+            </p>
+            <p xml:id="bHY6yh-de2985-d1e180"> /
+             ¶ <ref xml:id="bHY6yh-de2985-Rd1e178">8 capitulo</ref> <quote xml:id="bHY6yh-de2985-Qd1e182">Non insole pa<lb ed="#B" break="no"/>ne 
+             viuit homo Sed ex omni verbo 
+             quod egreditur ex ore dei</quote>
+            </p>
+            <p xml:id="bHY6yh-de2985-d1e190">
+              <lb ed="#B"/>
+                <ref xml:id="bHY6yh-de2985-Rd1e192">Psalmista 102 psalmus</ref> <quote xml:id="bHY6yh-de2985-Qd1e194"> <!--line jumps up-->Iustitia <!--line jumps down--> domini in fili<lb ed="#B" break="no"/>os
+                filiorum his qui seruant testa<lb ed="#B" break="no"/>mentum 
+                eius et memores sunt man<lb ed="#B" break="no"/>datorum 
+                ipsius ad faciendum ea. </quote>
+            </p>
+            <p xml:id="bHY6yh-de2985-d1e204"> /
```

Comment:
careful with this "slash". Was this left over from the venice edition. If so, make sure to delete.

I see a lot of these slashes right after the paragraph tag. I wonder if they should be deleted?